Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -225,6 +225,8 @@ ul.DropDownMenuOptions li a > span:last-of-type
   grid-area: span;
   align-self: center;
   padding-left: var(--remotion-themed-spacing);
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
How does the effect look in NovaGray?

Reply:
ATM the ellipsis is not added for NovaGray, only the alignment - the popup overflows the viewport when too large. Now I'll have to change it anyway when adding the margin, as it would look weird otherwise